internal/request/graphql/schema/types/commits.go

```diff
@@ -136,6 +136,55 @@ func CommitLinkObject() *gql.Object {
 	})
 }
 
+func CommitsFilterFieldNameArg() *gql.InputObject {
+	return gql.NewInputObject(gql.InputObjectConfig{
+		Name:        "CommitsFieldNameFilterArg",
+		Description: "Filter operators for commit fieldName.",
+		Fields: gql.InputObjectConfigFieldMap{
+			"_eq": &gql.InputObjectFieldConfig{
+				Description: eqOperatorDescription,
+				Type:        gql.String,
+			},
+			"_ne": &gql.InputObjectFieldConfig{
+				Description: neOperatorDescription,
+				Type:        gql.String,
+			},
+		},
+	})
+}
+
+func CommitsFilterArg(fieldNameFilter *gql.InputObject) *gql.InputObject {
+	var selfRefType *gql.InputObject
+
+	inputCfg := gql.InputObjectConfig{
+		Name:        "CommitsFilterArg",
+		Description: "Filter argument for commits query.",
```

Comment:
todo: Please declare this description in the same location as the other descriptions.